runtime/src/main/java/io/micronaut/jackson/JacksonConfiguration.java

```diff
@@ -357,4 +379,22 @@ public void setPropertyNamingStrategy(PropertyNamingStrategy propertyNamingStrat
         }
         return javaTypes.toArray(new JavaType[0]);
     }
+
+    @ConfigurationProperties("hateoas")
+    @Requires("jackson.hateoas")
+    public static final class Hateoas {
+        /**
+         * If true _embedded.errors will always be an array. When set to false, _embedded.errors will be serialized
+         * as an object for 1 error or serialized as an array for 2 or more errors.
+         */
+        private boolean alwaysSerializeErrorsAsList = false;
+
+        public boolean isAlwaysSerializeErrorsAsList() {
+            return alwaysSerializeErrorsAsList;
+        }
+
+        public void setAlwaysSerializeErrorsAsList(boolean alwaysSerializeErrorsAsList) {
+            this.alwaysSerializeErrorsAsList = alwaysSerializeErrorsAsList;
+        }
+    }
```

Comment:
I don't think we need a whole new nested config props just for this. Adding the field at the top level is fine